Sofascore/Sofascore/Views/HeaderCustomView.swift

```diff
@@ -0,0 +1,71 @@
+import Foundation
+import SwiftUI
+import SofaAcademic
+import SnapKit
+
+class HeaderCustomView: BaseView {
+    
+    private let logo: UIImageView = .init()
+    private let horizontalStack: UIStackView = .init()
+    private let cupImage: UIImageView = .init()
+    private let settings: UIImageView = .init()
+    var delegate: HeaderCustomViewDelegate?
+    
+    override func addViews() {
+        addSubview(logo)
+        addSubview(horizontalStack)
+        horizontalStack.addArrangedSubview(cupImage)
+        horizontalStack.addArrangedSubview(settings)
+    }
+    
+    override func setupConstraints() {
+        logo.snp.makeConstraints {
+            $0.top.bottom.equalToSuperview().inset(14)
+            $0.leading.equalToSuperview().inset(16)
+            $0.trailing.lessThanOrEqualTo(horizontalStack.snp.leading)
+        }
+        horizontalStack.snp.makeConstraints {
+            $0.top.bottom.equalToSuperview()
+            $0.trailing.equalToSuperview().inset(16)
+        }
+    }
+    
+    @objc private func settingsTapped() {
+        delegate?.settingsImageTapped()
+    }
+    
+    override func styleViews() {
+        backgroundColor = Colors.colorPrimaryDefault
+        horizontalStack.spacing = 16
+        logo.image = UIImage(named: "sofascoreLockup")
+        logo.contentMode = .scaleAspectFill
+        cupImage.image = UIImage(named: "icon2")
+        cupImage.contentMode = .scaleAspectFit
+        settings.image = UIImage(named: "icon1")
+        settings.contentMode = .scaleAspectFit
+        settings.isUserInteractionEnabled = true
+        settings.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(settingsTapped)))
+    }
+}
+
+protocol HeaderCustomViewDelegate: AnyObject {
+    func settingsImageTapped()
+}
```

Comment:
```suggestion
protocol HeaderCustomViewDelegate: AnyObject {

    func settingsImageTapped()
}
```

A moze i u zaseban file